tutor/src/screens/assignment-builder/builder/tasking.js

```diff
@@ -127,79 +129,84 @@ class Tasking extends React.Component {
   }
 
   render() {
-    if (!this.taskings.length) { return null; }
```

Comment:
I'm guessing this got removed because you need it to render something (the `<Col>`) even when tasking is null?

Reply:
yep, if it's null we still need to render the checkbox so it can be re-selected